src/arche/arche.py

```diff
@@ -242,7 +242,7 @@ def run_comparison_rules(self):
             return
         self.save_result(
             coverage_rules.compare_scraped_fields(
-                self.source_items.df, self.target_items.df
+                self.source_items.df.columns.values, self.target_items.df.columns.values
```

Comment:
Using _df.columns.values_ is not recommended:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Index.values.html

better is to use:

> Warning We recommend using Index.array or Index.to_numpy(), depending on whether you need a reference to the underlying data or a NumPy array.

Reply:
In this case maybe it's not important and this will work.

Reply:
@manycoding Considering [my comment below](https://github.com/scrapinghub/arche/pull/43/files#r271598600) the expression to get the column diff will be
```
df1.columns.difference(df2.columns)
```
so you can keep
```
coverage_rules.compare_scraped_fields(self.source_items.df, self.target_items.df)
```
 signature.